src/main/java/sirius/db/es/ElasticQuery.java

```diff
@@ -120,7 +120,10 @@
      * This timeout / TTL keeps the PIT open. Using a high time is not a problem here, because it gets closed after the
      * stream is consumed, so we rather try to choose a high timeout to avoid issues with prematurely closed PITs.
      */
-    public static final String STREAM_BLOCKWISE_PID_TTL = "30m";
+    private static final String STREAM_BLOCKWISE_PID_TTL = "30m";
```

Comment:
Is `STREAM_BLOCKWISE_PID_TTL` a typo and should rather be `STREAM_BLOCKWISE_PIT_TTL`?

Reply:
thx, fixed